index.html

```diff
@@ -326,7 +326,7 @@ <h3 id="server-performance"><a href="#server-performance">Server performance</a>
             <td class="ok"><a href="https://www.pulsesecure.net/download/techpubs/current/1027/Pulse-vADC-Solutions/Pulse-Virtual-Traffic-Manager/17.4/ps-vtm-17.4-userguide.pdf#page=175">yes</a></td>
             <td class="ok"><a href="https://www.pulsesecure.net/download/techpubs/current/1027/Pulse-vADC-Solutions/Pulse-Virtual-Traffic-Manager/17.4/ps-vtm-17.4-userguide.pdf#page=219">yes</a></td>
             <td class="ok"><a href="https://www.pulsesecure.net/download/techpubs/current/1027/Pulse-vADC-Solutions/Pulse-Virtual-Traffic-Manager/17.4/ps-vtm-17.4-userguide.pdf#page=175">yes</a></td>
-            <td class="alert">no</td>
+            <td class="ok"><a href="https://www.pulsesecure.net/download/techpubs/current/1441/Pulse-vADC-Solutions/Pulse-Virtual-Traffic-Manager/18.3/ps-vtm-18.3-userguide.pdf#page=226">yes</a></td>
```

Comment:
"The Traffic Manager also supports TLSv1.3, but only for connections to virtual servers.." 

For my own education, could you explain what "virtual server" is in this context?

Reply:
Hi Ilya,

Sure. The product is a middlebox proxy (like HAproxy, Varnish, nginx, F5, etc) and virtual server is the code that accepts a user's TLS connection and decrypts it. Since we're just a small team of 3, and it's our own TLS stack, we've only had time to add server-socket support to decrypt TLS 1.3 connections, and not make outgoing (to the backend webservers) TLS 1.3 connections. That will be next release.

Would you prefer "incoming only", "partial" or similar wording, or wait for another 3 months then I can make the cell a full "ok"?

Cheers,
Jonathan.